src/connectors/panoptic/panoptic.controllers.ts

```diff
@@ -837,14 +960,17 @@ export async function getAccountPremium(
     req.atTick,
     req.isLong
   );
-  return liquidity;
+  return {
+    other1: liquidity[0], 
```

Comment:
Let's rename other1 to premiumForToken0, and other2 to premiumForToken1

Reply:
Done: 6e29fb094bd2451b60e681862a7d1ba97608e214